docusaurus/src/remark/docsHeaderDecoration.js

```diff
@@ -0,0 +1,135 @@
+const fetch = require("node-fetch");
+const visit = require("unist-util-visit");
+
+const registry_url =
+  "https://connectors.airbyte.com/files/generated_reports/connector_registry_report.json";
+let registry = {};
+let loading = false;
+
+const plugin = () => {
+  const transformer = async (ast, vfile) => {
+    if (!isDocsPage(vfile)) return;
+
+    await fetchCatalog();
+
+    const pathParts = vfile.path.split("/");
+    const connectorName = pathParts.pop().split(".")[0];
+    const connectorType = pathParts.pop();
+    const dockerRepository = `airbyte/${connectorType.replace(
+      /s$/,
+      ""
+    )}-${connectorName}`;
+    const registryEntry = registry.find(
+      (r) => r.dockerRepository_oss === dockerRepository
+    );
+
+    if (!registryEntry) return;
+
+    visit(ast, "heading", (node) => {
+      let headerFound = false;
+      if (node.depth === 1 && node.children.length === 1 && !headerFound) {
+        headerFound = true;
+        const originalTitle = node.children[0].value;
+        const originalId = node.data.hProperties.id;
+
+        node.type = "html";
+        node.children = undefined;
+        node.value = buildConnectorHTMLContent(
+          registryEntry,
+          originalTitle,
+          originalId
+        );
+      }
+    });
+  };
+  return transformer;
+};
+
+const buildConnectorHTMLContent = (
+  registryEntry,
+  originalTitle,
+  originalId
+) => {
+  const htmlContent = `
+  <h1 id="${originalId}">${originalTitle}</h1>
+  <small>
+    <table>
+      <tbody>
+        <tr>
+          <td rowSpan="4">
+            <img style="max-height: 75px" src="${registryEntry.iconUrl_oss}" />
+          </td>
+          <td>Support Level: </td>
+          <td><strong>${capitalizeFirstLetter(
+            registryEntry.supportLevel_oss
+          )}</strong></td>
+        </tr>
+        <tr>
+          <td>Definition Id: </td>
+          <td>${registryEntry.definitionId}</td>
+        </tr>
+        <tr>
+          <td>Latest Version: </td>
+          <td>${registryEntry.dockerImageTag_oss}</td>
+        </tr>
+        <tr>
+          <td>Availability:</td>
+          <td>Airbyte Cloud: ${
+            registryEntry.is_cloud ? "✅" : "❌"
+          }  |  Airbyte OSS: ${registryEntry.is_oss ? "✅" : "❌"}</td>
+        </tr>
+      </tbody>
+    </table>
+  </small>
+  <br />
```

Comment:
Manual line breaks are not good HTML practice. The `small` or `table` should just have a classname with a margin-bottom if that's what we're trying to achieve here.